### PR TITLE
fine-tuned inclusion of target node ids if target node is not part of AMps2LionWebConverter

### PR DESCRIPTION
**Pull request by enikao**

### CODE REVIEW

File: docs/reference/converter-lang.adoc [hunk 1]
```diff
@@ -72,6 +72,25 @@ closure::
 Converts the listed nodes, all descendants, and all referenced nodes.
 We apply this strategy until we include all Annotation, Containment, and Reference targets. 
 
+
+.Inclusion of nodes and ids
```
Comment (dslmeinte) [inline docs/reference/converter-lang.adoc:76]: I still find this hard to grok. A diagram might help, but I think that I should make an effort myself, then :)

File: solutions/io.lionweb.mps.json/models/io.lionweb.mps.json.instance.mps2lionweb.mps [hunk 2]
```diff
@@ -3619,6 +3624,25 @@
...
+      <node concept="3clFbT" id="3i3chebGTNB" role="33vP2m" />
+      <node concept="z59LJ" id="3i3chebGVPG" role="lGtFl">
+        <node concept="TZ5HA" id="3i3chebGVPH" role="TZ5H$">
+          <node concept="1dT_AC" id="3i3chebGVPI" role="1dT_Ay">
+            <property role="1dT_AB" value="If true, exports parent node ids, even if parent is not converted." />
+          </node>
+        </node>
+        <node concept="TZ5HA" id="3i3chebGW5R" role="TZ5H$">
+          <node concept="1dT_AC" id="3i3chebGW5S" role="1dT_Ay">
+            <property role="1dT_AB" value="Defaults to false." />
```
Comment (dslmeinte) [inline solutions/io.lionweb.mps.json/models/io.lionweb.mps.json.instance.mps2lionweb.mps]: I thought we agreed that defaulting to exporting parent node IDs (even if unconverted) was the sensible default?

**Review by dslmeinte — CHANGES_REQUESTED** [review]

File: docs/reference/converter-lang.adoc (hunk 1 above)
Comment (enikao) [reply docs/reference/converter-lang.adoc:76]: Removed the node, as it's not relevant in default settings.

I'd still happily receive a better explanation (-:

File: solutions/io.lionweb.mps.json/models/io.lionweb.mps.json.instance.mps2lionweb.mps (hunk 2 above)
Comment (enikao) [reply solutions/io.lionweb.mps.json/models/io.lionweb.mps.json.instance.mps2lionweb.mps]: You're right, fixed it (and added tests).

**Review by dslmeinte — APPROVED** [review]

**Pull request merged**